subsys/testsuite/ztest/include/zephyr/ztest_test_new.h

```diff
@@ -263,7 +263,7 @@ void ztest_test_pass(void);
 void ztest_test_skip(void);
 
 #define Z_TEST(suite, fn, t_options, use_fixture)                                                  \
-	struct ztest_unit_test_stats UTIL_CAT(z_ztest_unit_test_stats_, fn);      \
+	struct ztest_unit_test_stats Z_CAT5(z_ztest_unit_test_stats_, suite, _, fn, );             \
```

Comment:
should not be using "internal" macros. please consider using `UTIL_CAT` or adding a new "public" macro for this purpose.

Reply:
Can we use `##` here to be consistent with the rest of the macro?

Reply:
yeah, should have used the `##`...